tech-audit.js

```diff
@@ -268,6 +269,10 @@
     clientSideListAdd("Twitter Widgets");
   }
 
+  if (elementExists('[data-reactroot]') || elementExists('[data-reactid]')) {
+    clientSideListAdd("React");
+  }
```

Comment:
I haven't tested extensively, but the browser extension claims to support all versions. I tested it in a few sites built with recent. Can do more testing if useful.